IdentityCore/src/cache/crypto/MSIDAssymetricKeyLookupAttributes.m

```diff
@@ -25,46 +25,39 @@
 
 @implementation MSIDAssymetricKeyLookupAttributes
 
+/*
+ This particular query generates the asymmetric key pair but only saves the private key in the keychain.
+ Please refer to https://developer.apple.com/documentation/security/certificate_key_and_trust_services/keys/generating_new_cryptographic_keys?language=objc
+ */
 - (NSDictionary *)assymetricKeyPairAttributes
 {
-    NSMutableDictionary *keyPairAttr = [NSMutableDictionary new];
-    keyPairAttr[(__bridge id)kSecAttrIsPermanent] = @YES;
-    keyPairAttr[(__bridge id)kSecAttrKeyType] = (__bridge id)kSecAttrKeyTypeRSA;
-    keyPairAttr[(__bridge id)kSecAttrKeySizeInBits] = @2048;
-    keyPairAttr[(__bridge id)kSecAttrLabel] = self.keyDisplayableLabel;
-    keyPairAttr[(__bridge id)kSecAttrAccessible] = (__bridge id)kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly;
+    NSData *tag = [self.privateKeyIdentifier dataUsingEncoding:NSUTF8StringEncoding];
```

Comment:
What about label, accessible, sensitive and extractable attributes? Those seem to be gone.

Reply:
Addressed. Added them back